scripts/runTestsForChangedUtils.bash

```diff
@@ -0,0 +1,116 @@
+testsToRun=""
```

Comment:
You don't really need to do this manually. `jest --onlyChanged --silent` already does that.

Reply:
Does hardhat not have the same feature?